src/components/Dropdown/Dropdown.jsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import propTypes from 'prop-types';
+import classes from 'classnames';
+import Icon from '../Icon/Icon';
+/**
+ * The dropdown menu allow user to customize the button text, icon and action.
+ */
+const Dropdown = ({ className, items, topIcon, topText, open }) => {
+  const dropdownClass = classes('Dropdown', { 'Dropdown--open': open }, className);
+  return (
+    <div className={dropdownClass}>
+      <button type="button" className="Dropdown--button" tabIndex="0">
+        {topIcon && <Icon width={16} height={16} kind={topIcon} />}
+        <p>{topText}</p>
+        <Icon className="Dropdown--arrow" width={16} height={16} kind="chevron-down" />
+      </button>
+      <div className="Dropdown--container">
+        <ul className="Dropdown--list">
+          {items.map(item => (
+            <li key={item.value}>
+              <button onClick={item.onClick} type="button" className="Dropdown--item" tabIndex="0">
```

Comment:
add these props for `item `
```js
{
 /* type of DOM element to render (defaults to link)*/
 type: propTypes.oneOf(['button', 'link']), 
/* if link, location of link */
 href: propTypes.string
}
```
then conditionally render a button or anchor tag depending on the `item.type` 

This way if we want to have this be a context menu for navigation inside the app we can use buttons but if not it will have good a11y with links